src/app/grid/grid-groupby-summary-sample/grid-groupby-summary-sample.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
```

Comment:
The **grid-groupby-summary-sample.component.spec.ts** file can be removed, since it has no actual purpose.

Reply:
Done.